medical_pathology_import/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===============================
+Odoo Medical Pathology Importer
+===============================
+
+This module adds a wizard and interface for the import of external Pathology
+data.
+
+Usage
+=====
+
```

Comment:
Improve this, add where the wizard is & how to use.

Also maybe add a note of how to implement when creating adapters